src/components/FilterBarOptions.tsx

```diff
@@ -2,7 +2,7 @@ import { useState, useCallback } from "react";
 import { IFilter } from "../interfaces/IFilter";
 import { determineClassName } from "../utils/FilterHelpers";
 
-export interface IFilterBarOptionsProps {
+interface IFilterBarOptionsProps {
```

Comment:
Thank you for these clean-ups. I often export the props interfaces out of habit despite rarely needing to do so.